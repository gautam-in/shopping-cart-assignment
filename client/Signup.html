<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Sabka Bazar</title>
    <link rel="stylesheet" href="style/design.css" />
  </head>

  <body>
    <header id="main-header" class="main-header">
      <uc-header count="0"></uc-header>
    </header>
    <div class="loginForm">
      <div class="column">
        <h2>Signup</h2>
        <p>We do not share your personal details with anyone.</p>
      </div>
      <div class="column">
        <div class="error-block"></div>
        <form onsubmit="SubmitForm(event)">
          <input type="text" id="firstName" placeholder="First Name" required />
          <input type="text" id="lastName" placeholder="Last Name" required />
          <input type="email" id="email" placeholder="Email" required />
          <input
            type="password"
            id="password"
            placeholder="Password"
            required
          />
          <input
            type="password"
            id="confirmPassword"
            placeholder="Confirm Password"
            required
          />
          <button type="submit">Signup</button>
        </form>
      </div>
    </div>
    <footer>
      <uc-footer></uc-footer>
    </footer>
  </body>

  <script defer src="./header.js"></script>
  <script defer src="./footer.js"></script>
  <script defer>
    function SubmitForm(e) {
      const pass = document.getElementById("password").value;
      const cPass = document.getElementById("confirmPassword").value;
      if (pass !== cPass) {
        document.querySelector(".error-block").innerText =
          "Password does not matched!!";
      } else {
        document.querySelector(".error-block").innerText = "";
        alert("Data submitted successfully");
      }
      e.preventDefault();
    }
  </script>
</html>
