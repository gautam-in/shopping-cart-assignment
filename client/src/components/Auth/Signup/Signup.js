import { useHistory } from "react-router-dom";
import "./signup.scss";
import Input from "../../UI/Input/Input";

const Signup = () => {
  const history = useHistory();
  const submitHandler = (e) => {
    if (e.target[3].value === e.target[4].value) {
      e.preventDefault();
      history.push("/");
    } else {
      alert("Password does not match");
      e.preventDefault();
    }
  };
  return (
    <div className="flex--row">
      <div className="pageDetails">
        <h1 className="pageDetails__heading">SignUp</h1>
        <p className="pageDetails__desc">We do not share your personal details with anyone</p>
      </div>
      <div className="auth-form">
        <form onSubmit={submitHandler} method="POST" action="/">
          <div className="rowInput flex--column">
            <Input
              type="text"
              placeholder="First Name"
              className="rowInput__inputText"
              id="firstName"
              required={true}
              name="firstName"
            />
            <label htmlFor="firstName" className="rowInput__labelHelper">
              First Name
            </label>
          </div>
          <div className="rowInput flex--column">
            <Input
              type="text"
              placeholder="Last Name"
              className="rowInput__inputText"
              id="lastName"
              required
              name="lastName"
            />
            <label htmlFor="lastName" className="rowInput__labelHelper">
              Last Name
            </label>
          </div>
          <div className="rowInput flex--column">
            <Input
              type="email"
              placeholder="Email"
              className="rowInput__inputText"
              id="email"
              required
              name="email"
            />
            <label htmlFor="email" className="rowInput__labelHelper">
              Email
            </label>
          </div>
          <div className="rowInput flex--column">
            <Input
              type="password"
              name="password"
              id="password"
              className="rowInput__inputText"
              placeholder="Password"
              required
            />
            <label htmlFor="password" className="rowInput__labelHelper">
              Password
            </label>
          </div>
          <div className="rowInput flex--column">
            <Input
              type="password"
              name="confirmPassword"
              id="confirmPassword"
              className="rowInput__inputText"
              placeholder="Confirm Password"
              required
            />
            <label htmlFor="confirmPassword" className="rowInput__labelHelper">
              Confirm Password
            </label>
          </div>
          <div className="submit-btn">
          <input className="submit" type="submit" value="SignUp" />
          </div>
         
        </form>
      </div>
    </div>
  );
};

export default Signup;