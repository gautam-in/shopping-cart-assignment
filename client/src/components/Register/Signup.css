.textfield{
    width:50%;
}
.btn{
    margin-top: 20px;
}
.confirmPassErr{
    color:red;
    margin: 0;
    font-size: 0.75rem;
    margin-top: 3px;
    text-align: left;
    font-family: "Roboto", "Helvetica", "Arial", sans-serif;
    font-weight: 400;
    line-height: 1.66;
    letter-spacing: 0.03333em;
}
.parentRegisterContainer{
    margin-top: 20px!important;
}