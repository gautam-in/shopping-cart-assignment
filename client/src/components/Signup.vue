<template>
  <section id="signupContainer">
    <div class="container">
      <div class="row">
        <div class="signupInfo col-lg-6 col-xl-6 col-md-6 col-sm-12 col-12">
          <h2>Signup</h2>
          <p>We do not share your personal details with anyone</p>
        </div>
        <div class="col-lg-6 col-xl-6 col-md-6 col-sm-12 col-12">
          <div class="mb-3 row">
            <label
              for="firstName"
              class="col-sm-2 col-lg-3 col-xl-3 col-md-3 col-form-label"
              >First Name</label
            >
            <div class="col-lg-4 col-xl-7 col-md-4 col-sm-10">
              <input
                v-model="$v.user.firstName.$model"
                type="text"
                class="form-control"
                id="firstName"
              />
              <div
                class="error"
                v-if="$v.user.firstName.$dirty && !$v.user.firstName.required"
              >
                First Name is required.
              </div>
            </div>
          </div>
          <div class="mb-3 row">
            <label
              for="lastName"
              class="col-sm-2 col-lg-3 col-xl-3 col-md-3 col-form-label"
              >Last Name</label
            >
            <div class="col-lg-4 col-xl-7 col-md-4 col-sm-10">
              <input
                v-model="$v.user.lastName.$model"
                type="text"
                class="form-control"
                id="lastName"
              />
              <div
                class="error"
                v-if="$v.user.lastName.$dirty && !$v.user.lastName.required"
              >
                Last Name is required.
              </div>
            </div>
          </div>
          <div class="mb-3 row">
            <label
              for="email"
              class="col-sm-2 col-lg-3 col-xl-3 col-md-3 col-form-label"
              >Email</label
            >
            <div class="col-lg-4 col-xl-7 col-md-4 col-sm-10">
              <input
                v-model="$v.user.email.$model"
                type="text"
                class="form-control"
                id="email"
              />
              <div
                class="error"
                v-if="$v.user.email.$dirty && !$v.user.email.required"
              >
                Email is required.
              </div>
            </div>
          </div>
          <div class="mb-3 row">
            <label
              for="inputPassword"
              class="col-sm-2 col-lg-3 col-xl-3 col-md-3 col-form-label"
              >Password</label
            >
            <div class="col-lg-4 col-xl-7 col-md-4 col-sm-10">
              <input
                v-model="$v.user.password.$model"
                type="password"
                class="form-control"
                id="inputPassword"
              />
              <div
                class="error"
                v-if="$v.user.password.$dirty && !$v.user.password.required"
              >
                Password is required.
              </div>
            </div>
          </div>
          <div class="mb-3 row">
            <label
              for="confiemPassword"
              class="col-sm-2 col-lg-3 col-xl-3 col-md-3 col-form-label"
              >Confirm Password</label
            >
            <div class="col-lg-4 col-xl-7 col-md-4 col-sm-10">
              <input
                v-model="$v.user.confirmPassword.$model"
                type="password"
                class="form-control"
                id="confirmPassword"
              />
              <div
                class="error"
                v-if="
                  $v.user.confirmPassword.$dirty &&
                  !$v.user.confirmPassword.sameAsPassword
                "
              >
                Password is different.
              </div>
            </div>
          </div>

          <div class="mb-3 row">
            <div
              class="btnContainer col-lg-10 col-xl-12 col-md-4 col-sm-12 col-12"
            >
              <button
                @click="register()"
                :disabled="$v.$error || $v.user.$anyError"
                class="btnStyle"
                :class="$v.$error || $v.user.$anyError ? 'op07' : ''"
              >
                Signup
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</template>

<script>
import axios from "axios";
import { required, sameAs } from "vuelidate/lib/validators";

export default {
  name: "Signup",
  data() {
    return {
      user: {
        firstName: "",
        lastName: "",
        email: "",
        password: "",
        confirmPassword: "",
      },
    };
  },
  validations: {
    user: {
      firstName: {
        required,
      },
      lastName: {
        required,
      },
      email: {
        required,
      },
      password: {
        required,
      },
      confirmPassword: {
        required,
        sameAsPassword: sameAs("password"),
      },
    },
  },
  methods: {
    register() {
      axios.post("http://localhost:5000/register", this.user).then((res) => {
        if (res.data.status == "success") {
          this.$router.push("/signin");
        }
      });
    },
  },
};
</script>

<style scoped>
p {
  font-size: 15px;
  margin-top: 1.5rem;
}

.col-form-label {
  text-align: initial;
}

.btnContainer {
  padding-left: 3.8rem;
}

button {
  width: 62%;
}
</style>
