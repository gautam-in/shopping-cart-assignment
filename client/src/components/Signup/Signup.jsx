import { useState } from 'react';

const formInitialValues = {
  firstName: '',
  lastName: '',
  email: '',
  password: '',
  confirmPassword: ''
};

const Signup = () => {
  const [formValues, setFormValues] = useState(formInitialValues);
  const { firstName, lastName, email, password, confirmPassword } = formValues;

  const handleChange = (e) => {
    const { name, value } = e.target;
    setFormValues((prev) => {
      return { ...prev, [name]: value };
    });
  };

  const handleSubmit = (e) => {
    e.preventDefault();
    if (password !== confirmPassword) {
      alert('Password mismatch!');
      return;
    }
    setFormValues(formInitialValues);
    alert('SignUp Successfull');
  };

  return (
    <div className="flex-container">
      <div className="center-box">
        <h1>Signup</h1>
        <p>We do not share your personal details with anyone.</p>
      </div>
      <div className="center-box">
        <form method="post" action="/" onSubmit={handleSubmit}>
          <div className="group">
            <label className="form-input-label">First Name</label>
            <input
              type="text"
              className="form-input"
              required
              name="firstName"
              value={firstName}
              onChange={handleChange}
            />
          </div>
          <div className="group">
            <label className="form-input-label">Last Name</label>
            <input
              type="text"
              className="form-input"
              required
              name="lastName"
              value={lastName}
              onChange={handleChange}
            />
          </div>
          <div className="group">
            <label className="form-input-label">Email</label>
            <input
              type="email"
              className="form-input"
              required
              name="email"
              value={email}
              onChange={handleChange}
            />
          </div>
          <div className="group">
            <label className="form-input-label">Password</label>
            <input
              type="password"
              className="form-input"
              required
              name="password"
              value={password}
              onChange={handleChange}
              minLength="6"
              pattern="^[a-zA-Z0-9]*$"
            />
          </div>
          <div className="group">
            <label className="form-input-label">Confirm Password</label>
            <input
              type="password"
              className="form-input"
              required
              name="confirmPassword"
              value={confirmPassword}
              onChange={handleChange}
              minLength="6"
              pattern="^[a-zA-Z0-9]*$"
            />
          </div>
          <button className="form-btn" type="submit">
            Signup
          </button>
        </form>
      </div>
    </div>
  );
};

export default Signup;
