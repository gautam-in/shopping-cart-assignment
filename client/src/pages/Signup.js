import React, { useEffect, useRef, useState } from "react";
import { useDispatch } from "react-redux";
import { LoginActions } from "../store/login-slice";

import { useHistory } from "react-router-dom";
import "./Login-signup.css";

function Signup() {
  let dispatch = useDispatch();

  let passwordRef = useRef();
  let cpasswordRef = useRef();
  let userEmail = useRef();

  let history = useHistory();

  let [isPassMatching, setisPassMatching] = useState(false);

  useEffect(() => {
    setisPassMatching(true);
  }, []);

  let validateSignUp = (event) => {
    event.preventDefault();
    let password = passwordRef.current.value;
    let cpassword = cpasswordRef.current.value;

    if (password.trim() !== cpassword.trim()) {
      setisPassMatching(false);
      return;
    }
    setisPassMatching(true);

    localStorage.setItem("userEmail", userEmail.current.value.trim());
    localStorage.setItem("userPass", passwordRef.current.value.trim());
    dispatch(LoginActions.setisLoginStatus(true));
    history.replace("/Home");
  };

  return (
    <section className="main-container">
      <aside className="aside signup-container">
        <h1>Sign-up</h1>
        <p>Get access to your Orders, Wishlist and Recommendations!</p>
      </aside>
      <main className="login-form signup-container">
        <form onSubmit={validateSignUp}>
          <label htmlFor="fname">First Name</label>
          <input
            className="input"
            type="text"
            name="fname"
            id="fname"
            //placeholder="First Name"
            title="enter the first name here."
            required
            autoFocus
          />
          <label htmlFor="lname">Last Name</label>
          <input
            className="input"
            type="text"
            name="lname"
            id="lname"
            title="enter the last name here."

            //placeholder="Last Name"
          />
          <label htmlFor="email">Email</label>
          <input
            className="input"
            type="email"
            name="email"
            id="email"
            //placeholder="email"
            title="enter the email address here."
            required
            ref={userEmail}
          />
          <label htmlFor="password">Password</label>
          <input
            className="input"
            type="password"
            name="password"
            id="password"
            //placeholder="password"
            required
            pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}"
            title="Must contain at least one number and one uppercase and lowercase letter, and at least 8 or more characters"
            ref={passwordRef}
          />

          <label htmlFor="cpassword">Confirm Password</label>
          <input
            className="input"
            type="password"
            name="cpassword"
            id="cpassword"
            //placeholder="confirm password"
            required
            pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}"
            title="Must contain at least one number and one uppercase and lowercase letter, and at least 8 or more characters"
            ref={cpasswordRef}
          />

          {!isPassMatching && (
            <p className="inputErrorMessage">
              password and confirm password does not match!
            </p>
          )}
          <input className="input" type="submit" value="Sign-up" />
        </form>
      </main>
    </section>
  );
}

export default Signup;
