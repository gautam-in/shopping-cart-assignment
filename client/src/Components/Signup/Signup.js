import React, { useEffect } from 'react'
import { SignupWrapper } from './Signup.style';
import InputFormWrapper from '../Form/InputForm';
import { useSelector } from 'react-redux';
import { useNavigate } from 'react-router-dom'


const Signup = ({ handleAction, title }) => {

    let navigate = useNavigate();
    const description = 'We donot share personal details with anyone';
    const authToken = useSelector(state => state.SIGNUPREDUCER.currentUser?.stsTokenManager?.refreshToken);

    useEffect(()=>{
        if (authToken) {
            navigate('/')
        }
    },[])

  return (
    <SignupWrapper>
        <InputFormWrapper 
            handleAction={handleAction} 
            title={title} 
            description={description}
            from="signup"
        />
    </SignupWrapper>
  )
}

export default Signup
