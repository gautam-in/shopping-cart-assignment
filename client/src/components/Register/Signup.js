import React, { useState } from "react";
import TextField from '@material-ui/core/TextField';
import Button from '@material-ui/core/Button';
import { Grid } from "@material-ui/core";
import { makeStyles } from '@material-ui/core/styles';
import Header from '../Header/Header';
import './Signup.css';
const data = [{
    id: 'firstname',
    label: 'FirstName',
    value: "",
    regex: /^[a-zA-z]+([\s][a-zA-Z]+)*$/,
    error: false,
    helperText: "please enter valid firstname"
}, {
    id: 'lastname',
    label: 'LastName',
    value: "",
    regex: /^[a-zA-z]+([\s][a-zA-Z]+)*$/,
    error: false,
    helperText: "please enter valid lastname"
},
{
    id: 'email',
    label: 'Email',
    value: "",
    regex: /^\w+@[a-zA-Z_]+?\.[a-zA-Z]{2,3}$/,
    error: false,
    helperText: "please enter valid email"
},
{
    id: 'password',
    label: 'Password',
    value: "",
    regex: /[0-9a-zA-Z]{6,}/,
    error: false,
    helperText: "please enter valid password"
},
{
    id: 'confirmPassword',
    label: 'Confirm Password',
    value: "",
    regex: /[0-9a-zA-Z]{6,}/,
    error: false,
    helperText: "please enter valid confirm password"
}
];
// import Button from '@material-ui/core/Button';
const useStyles = makeStyles((theme) => ({
    leftAlign: {
        display: 'flex',
        justifyContent: 'flex-end',
        [theme.breakpoints.down('xs')]: {
            justifyContent: 'center',
        },
        [theme.breakpoints.down('sm')]: {
            justifyContent: 'center'
        },

    },
    rightText: {
        width: '50%',
        [theme.breakpoints.down('xs')]: {
            justifyContent: 'center',
            width: '100%',
            paddingLeft: '5px'
        },
        [theme.breakpoints.down('sm')]: {
            textAlign: 'center'
        },
    },
    registerBtn: {
        width: '50%',
        [theme.breakpoints.down('xs')]: {
            justifyContent: 'center',
            width: '100%',
        },
    },
    parentRegisterContainer: {
        marginTop: '10px',
        [theme.breakpoints.down('xs')]: {
            padding: '8px'
        },
    }

}));
const SignUp = (props) => {
    const [registerData, setRegisterData] = useState(data);
    const [confirmPassErr, setConfirmPassErr] = useState(false);
    const classes = useStyles();
    const handleChange = (index, event) => {
        setRegisterData([...registerData.slice(0, index),
        {
            ...registerData[index], value: event.target.value, error: registerData[index].regex.test(event.target.value) ? false : true
        },
        ...registerData.slice(index + 1)
        ]);
        setConfirmPassErr(false);
    }
    const handleRegister = () => {
        const findPasswordObj = registerData.find((datas) => datas.id === "password");
        const findConformPasswordObj = registerData.find((datas) => datas.id === "confirmPassword");
        if (findPasswordObj && findConformPasswordObj && findPasswordObj.value !== findConformPasswordObj.value) {
            setConfirmPassErr(true);
        }
        else {
            setConfirmPassErr(false);
            props.history.push('/login');
        }
    }

    const isSignUpBtnDisabled = registerData.some((data) => data.error);
    const filedNotEmpty = registerData.some((data) => data.value === "");

    return (
        <React.Fragment>
            <Header/>
            <Grid container spacing={8} className={classes.parentRegisterContainer}>
                <Grid item xs={12} md={6} sm={12}>
                    <div className={classes.leftAlign}>
                        <div>
                            <h1>Signup</h1>
                            <p>We do not share your personal details with anyone</p>
                        </div>
                    </div>
                </Grid>
                <Grid item xs={12} md={6} sm={12} className={classes.rightText}>
                    <div>
                        {registerData.map((items, index) => {
                            return (
                                <div style={{paddingBottom:'15px'}} key={index}>
                                    <TextField
                                        // className="textfield"
                                        inputProps={{ "data-testid": items.id }}
                                        className={classes.rightText}
                                        onChange={(event) => handleChange(index, event)}
                                        required
                                        id="standard-required"
                                        label={items.label}
                                        value={items.value}
                                        error={items.error}
                                        helperText={items.error ? items.helperText : ""}
                                    />
                                </div>
                            )
                        })}
                        {confirmPassErr ? <div>
                            <span className="confirmPassErr">Confirm password should match password</span>
                        </div>
                            : null}
                    </div>
                    <div className='btn'>
                        <Button data-testid="signup-submit" className={classes.registerBtn} onClick={handleRegister} variant="contained" color="secondary" disabled={filedNotEmpty || isSignUpBtnDisabled}>
                            Sign Up
                        </Button>
                    </div>
                </Grid>
            </Grid>
        </React.Fragment>
    )
}
export default SignUp;