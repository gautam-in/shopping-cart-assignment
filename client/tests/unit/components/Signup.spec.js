import { mount, createLocalVue, RouterLinkStub } from "@vue/test-utils";
import Signup from "@/components/Signup.vue";
import Vuelidate from "vuelidate";
import Vue from "vue";
Vue.use(Vuelidate);
import VueRouter from "vue-router";

const localVue = createLocalVue();
localVue.use({
  Signup,
});

describe("Signup", () => {
  it("snapshot", () => {
    const wrapper = mount(Signup, {
      localVue,
      data() {
        return {
          user: {
            firstName: "",
            lastName: "",
            email: "",
            password: "",
            confirmPassword: "",
          },
        };
      },
    });
    expect(wrapper).toMatchSnapshot();
    // expect(wrapper.contains("div")).toBe(true);
  });
});
