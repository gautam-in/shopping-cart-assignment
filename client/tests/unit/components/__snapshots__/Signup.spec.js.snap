// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signup snapshot 1`] = `
<section id="signupContainer">
  <div class="container">
    <div class="row">
      <div class="signupInfo col-lg-6 col-xl-6 col-md-6 col-sm-12 col-12">
        <h2>Signup</h2>
        <p>We do not share your personal details with anyone</p>
      </div>
      <div class="col-lg-6 col-xl-6 col-md-6 col-sm-12 col-12">
        <div class="mb-3 row"><label for="firstName" class="col-sm-2 col-lg-3 col-xl-3 col-md-3 col-form-label">First Name</label>
          <div class="col-lg-4 col-xl-7 col-md-4 col-sm-10"><input type="text" id="firstName" class="form-control">
            <!---->
          </div>
        </div>
        <div class="mb-3 row"><label for="lastName" class="col-sm-2 col-lg-3 col-xl-3 col-md-3 col-form-label">Last Name</label>
          <div class="col-lg-4 col-xl-7 col-md-4 col-sm-10"><input type="text" id="lastName" class="form-control">
            <!---->
          </div>
        </div>
        <div class="mb-3 row"><label for="email" class="col-sm-2 col-lg-3 col-xl-3 col-md-3 col-form-label">Email</label>
          <div class="col-lg-4 col-xl-7 col-md-4 col-sm-10"><input type="text" id="email" class="form-control">
            <!---->
          </div>
        </div>
        <div class="mb-3 row"><label for="inputPassword" class="col-sm-2 col-lg-3 col-xl-3 col-md-3 col-form-label">Password</label>
          <div class="col-lg-4 col-xl-7 col-md-4 col-sm-10"><input type="password" id="inputPassword" class="form-control">
            <!---->
          </div>
        </div>
        <div class="mb-3 row"><label for="confiemPassword" class="col-sm-2 col-lg-3 col-xl-3 col-md-3 col-form-label">Confirm Password</label>
          <div class="col-lg-4 col-xl-7 col-md-4 col-sm-10"><input type="password" id="confirmPassword" class="form-control">
            <!---->
          </div>
        </div>
        <div class="mb-3 row">
          <div class="btnContainer col-lg-10 col-xl-12 col-md-4 col-sm-12 col-12"><button class="btnStyle">
              Signup
            </button></div>
        </div>
      </div>
    </div>
  </div>
</section>
`;
